### PR TITLE
Refactor/blo 30 change contract addresses for polygon

### PR DESCRIPTION
**Pull request by ibrica**

### CODE REVIEW

File: packages/evm-adapter/src/constants.ts [hunk 1]
```diff
@@ -1,4 +1,12 @@
 import { BigNumber, utils } from 'ethers';
 
-export const contractAddress = '0x707Cad6EDB79eE9F64818B09F14A7Aa7f4910e93';
+// For compability with BE and FE, remove in future
+export const contractAddress = '0xa35e123aA4AFeD2E4957897232F2ABf73f34cfB0';
```
Comment (martinavagyan) [inline packages/evm-adapter/src/constants.ts:4]: don't you want to use this variable inside ContractAddresses ?
```TS
export enum ContractAddresses {
  POLYGON_MUMBAI = contractAddress,
  POLYGON_MAINNET = contractAddress,
}
```

File: packages/evm-adapter/src/utils.ts [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import { ContractAddresses } from './constants';
+
+export const getContractAddress = (chainId: number): string | undefined => {
+  switch (chainId) {
+    case 80001:
```
Comment (martinavagyan) [inline packages/evm-adapter/src/utils.ts:5]: I think making an enum for those ids would make it easier to identify

**Review by martinavagyan — APPROVED** [review]

File: packages/evm-adapter/src/constants.ts (hunk 1 above)
Comment (ibrica) [reply packages/evm-adapter/src/constants.ts]: Removed

File: packages/evm-adapter/src/utils.ts (hunk 2 above)
Comment (ibrica) [reply packages/evm-adapter/src/utils.ts:5]: Ok

**Pull request merged**